MAINTENANCE.md

```diff
@@ -0,0 +1,57 @@
+# Maintenance
+This document outlines basic procedures for maintenance of issues, PRs, and releases.
+
+## Issues and Pull Requests
+Issues and pull requests are at the core of hapi's approach to documentation and messaging to the broader community.  They are rigorously labeled, associated with releases, and assigned to collaborators responsible for their resolution.  Official discussions, release notes, announcements, and community support all generally take the form of issues.  Finally, the source of truth for our changelogs are also managed through issues.
+
+### Labels
+Labels are standardized through [organization defaults](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/managing-default-labels-for-repositories-in-your-organization).  Here are some notable ones:
+ - `support` - requests for clarification or help.
+ - `bug` - reports of unintended behavior.
+ - `security` - indication of security implications.
+ - `feature` - requests for new functionality.
+ - `release notes` - documentation of changes in major releases, sometimes accompanied by a migration guide.
+ - `documentation` - non-code related changes to the readme, API docs, or similar.
+ - `dependency` - changes to underlying or peer dependencies.
```

Comment:
I'd just leave out the "underlying or peer" part.

Reply:
```suggestion
 - `dependency` - changes to dependencies.
```